.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "Saas Procurement App",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "saas_app",
+	"workspaceFolder": "/workspace",
+
+	"settings": {
+		"python.pythonPath": "/usr/local/bin/python",
+		"python.languageServer": "Pylance",
+		"python.linting.enabled": true,
+		"python.linting.pylintEnabled": true,
+	},
+
+	"extensions": [
+		"ms-python.python",
+		"eamodio.gitlens",
+		"editorconfig.editorconfig"
+	],
+
```

Comment:
can we add `"remoteUser": "vscode"` in this file? Just because it might fix something that is broken below in the dockerfile with the user